docs/guides/development/upgrading/upgrade-guides/2025-11-10.mdx

```diff
@@ -0,0 +1,892 @@
+
+1. **Updating branding.** Clerk Billing was originally developed under the working name "Commerce", which is why `/commerce/` appeared in endpoints. This API version standardizes on "Billing" instead of the deprecated "Commerce". We've also updated some of the terminology, including renaming "payment sources" to "payment methods".
+1. **Making it clear whether data exists.** Sometimes a response type has different information depending on the request that prompted the response. Older API versions handle the absence of this information in inconsistent ways--sometimes it was set to the empty value (like `""` for strings or `0` for numbers), sometimes it was set to `null`, and sometimes it wasn't set at all. 2025-11-10 standardizes on the following semantics:
+   - If the response is trying to tell you that the data definitely doesn't exist, it will explicitly be set to `null`.
+   - If the response is not making any assertion about the existence of the data, it will not include the key at all.
+   - If the response is trying to tell you that the data is set to the empty value (like `""` for strings, or `0` for numbers) it will be set to that value.
+1. **Cleaning up deprecated fields.** As our API responses have evolved, we've left deprecated fields in the responses for backwards compatibility. This API version removes previously-deprecated fields.
+
+## Backend Response Type Changes
+
+### Payers
+
+<Properties>
+  - `organization_id`
+  - `string`
+
+  The `organization_id` field will now be explicitly `null` for user payers, instead of omitted.
+
+  ---
+
+  - `organization_name`
+  - `string`
+
+  The `organization_name` field will now be explicitly `null` for user payers, instead of omitted. It can still be omitted from certain endpoints that do not provide that data.
+
+  ---
+
+  - `user_id`
+  - `string`
+
+  The `user_id` field will now be explicity `null` for organization payers, instead of omitted.
+
+  ---
+
+  - `first_name`
+  - `string`
+
+  The `first_name` field will now be explicitly `null` for organization payers, instead of `""`. It can also be omitted from certain endpoints that do not provide that data.
+
+  ---
+
+  - `last_name`
+  - `string`
+
+  The `last_name` field will now be explicitly `null` for organization payers, instead of `""`. It can also be omitted from certain endpoints that do not provide that data.
+
+  ---
+
+  - `email`
+  - `string`
+
+  The `email` field will now be explicitly `null` for organization payers, instead of `""`. It can also be omitted from certain endpoints that do not provide that data.
+
+  ---
+
+  - `image_url`
+  - `string`
+
+  The `image_url` field will now be omitted from endpoints that do not provide that data, instead of returning `""`.
+
+  ---
+
+  - `created_at`
+  - `integer`
+
+  The `created_at` field will now be omitted instead of returning `0` when an endpoint does not provide that data.
```

Comment:
For any of these "field will now be omitted" props, are we ok with `integer` or is `integer | undefined` more accurate?

What is our standard?